src/main/java/org/EnumAssigment/Athlete.java

```diff
@@ -0,0 +1,69 @@
+package org.EnumAssigment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Athlete {
+
+    private final int athleteNumber;
+    private final String athleteName;
+
+    private final String countryCode;
+
+    private final int sessionTimeResult;
```

Comment:
this is a time variable, you can use Java's Duration object or name it sessionTimeInSeconds if you transform in seconds